EasyLearn/app/views/player/index.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= stylesheet_link_tag "player", "data-turbo-track": "reload" %>
+
+  <nav>  
+    <h3 >Logo</h3>
+    <div class="menu">
+      <p>My events</p>
+      <div class="user-option">
+        <p class="username">username</p>
+      </div>  
+    </div>
+  </nav>
+  <div class="content">
+    <video  controls class="main-video"></video>
+    <h3 class="title">Pilas y Colas en C</h3>
+    <div class="user">
+      <%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
+      <p>Diego Steve</p>
+    </div>
+    <section>
+      <p class="description">Other content by the owner that may interest you</p>
+      <div class="video-suggested">
+        <div class="extra-video">
+          <video src="" controls> </video>
+          <div class="user">
+            <%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
+            <p>Mildred Nataly</p>
+          </div>
+      	</div>  
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
+						<p>Mildred Nataly</p>
+					</div>
+				</div>
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
+						<p>El bicho siuu</p>
+					</div>
+				</div>
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
+						<p>Pedro Sanchez</p>
+					</div>
+				</div>
+    	</div>  
```

Comment:
Check the indentation of this part of your code (remember that the spacing must be 2)

Reply:
Fixed identation